### DISCUSSION TITLE
UV on a static site

### DISCUSSION THREAD

**Discussion by sudonym-online** [UseInterstellar/Interstellar#844] [Q&A] [answered]
hey interstellar, how do you guys run [ultraviolet](https://github.com/titaniumnetwork-dev/Ultraviolet) on a static website?

**Comment by 0mhx** [comment]
interstellar isnt static

**Reply by sudonym-online** [reply]
but how am I able to host on a static hosting website like vercel?

**Reply by 0mhx** [reply]
vercel isnt static

**Comment by rifting** [comment]
> hey interstellar, how do you guys run [ultraviolet](https://github.com/titaniumnetwork-dev/Ultraviolet) on a static website?

(brief) ultraviolet developer here
you just put it in your static files as normal and set the transport to the wisp server you wish to use

**Reply by sudonym-online** [reply]
where do I set the wisp server, is it on one of the uv files when I install it? Is there a wisp server that you recommend I use? ( sorry if I'm asking too many questions I am a noob when it comes to these things)

**Reply by 0mhx** [reply]
make sure baremux is included, and use this code
```js
window.connection ||= new BareMux.BareMuxConnection("/baremux/worker.js");
await connection.setTransport("/epoxy/index.mjs", [{ wisp: "wispurl" }])
```
there are a couple of public wisp servers such as `wss://tomp.app`.

**Comment by sudonym-online** [comment, marked as answer]
oh I actually found a guide that shows how to do this w/ vercel: [https://github.com/crllect/UV-on-vercel](https://github.com/crllect/UV-on-vercel)

**Reply by rifting** [reply]
> oh I actually found a guide that shows how to do this w/ vercel: https://github.com/crllect/UV-on-vercel

outdated UV. don't use it lol

**Reply by sudonym-online** [reply]
yeah i know im going use the updated version when i use it
